pyaerocom/aeroval/compute_fairmode_stats.py

```diff
@@ -0,0 +1,98 @@
+"""
+Functions for computing the FAIRMODE statistics
+"""
+
+from math import sqrt, isclose
+from typing import Tuple
+
+
+def _get_RMSU(mean: float, std: float, species: str) -> Tuple[float, float, float, float]:
+    species_values = dict(
+        concno2=dict(
+            UrRV=0.24,
+            RV=200,
+            alpha=0.2,
+        ),
+        vmro3=dict(
+            UrRV=0.18,
+            RV=120,
+            alpha=0.79,
+        ),
+        concpm10=dict(
+            UrRV=0.28,
+            RV=50,
+            alpha=0.13,
+        ),
+        concpm25=dict(
+            UrRV=0.28,
+            RV=25,
+            alpha=0.3,
+        ),
+    )
+
+    if species not in species_values.keys():
+        raise KeyError(f"Species {species} not in list {species_values.keys()}")
+
+    UrRV = species_values[species]["UrRV"]
+    RV = species_values[species]["RV"]
+    alpha = species_values[species]["alpha"]
+
+    in_sqrt = (1 - alpha**2) * (mean**2 + std**2) + alpha**2 * RV**2
+
+    return UrRV * sqrt(in_sqrt), UrRV, RV, alpha
+
+
+def _get_fairmode_sign(mod_std: float, obs_std: float, R: float) -> float:
+    if obs_std * sqrt(2 * (1 - R)) == 0:
+        return 1
```

Comment:
why `_get_fairmode_sign` instead of `_fairmode_sign`? ...
And docstring ....

Also, this is a good place for some asserts
``` python
def _get_fairmode_sign(mod_std: float, obs_std: float, R: float) -> int:
   assert obs_std >= 0, f"negative {obs_std=}"
   assert 0 <= R <= 1, f"out of range {R=}"
   if obs_std <= 0 or R <= 0 or R >= 1:
        return 1
```